modules/ROOT/pages/apikit-asyncapi-avro-support.adoc

```diff
@@ -0,0 +1,63 @@
+= Implementing and Validating Avro Schemas in APIkit for AsyncAPI Module
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+APIkit for AsyncAPI supports Avro, a data serialization system that helps you define the format for specifying the structure and encoding of messages. This module shows you the structure of messages in the design phase. APIkit for AsyncAPI also supports xref:studio::datasense-concept.adoc[DataSense] of messages with Avro schema.
+
+== Schema Definitions
+
+The Avro spec defines its format by giving the value of `schemaFormat` as `application/vnd.apache.avro`.
+
+The Avro spec can describe the schema inline:
+
+----
+publish:
+  summary: inform details about person
+  operationId: lightMeasured
+  message:
+    schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
+    payload:
+      name: lightMeasured
+      type: record
+      fields:
+        - name: lumens
+          type: int
+          example: 134
+        - name: sentAt
+          type: string
+          logicalType: timestamp-millis
+          default: 10
+          example: 1985-04-12T23:20:50.52Z
+----
+
+The Avro spec can also refer to an external document that describes the schema in the Avro format:
+
+----
+channels:
+ smartylighting.streetlights.1.0.event.{streetlightId}.monitor:
+   publish:
+     message:
+       schemaFormat: application/vnd.apache.avro;version=1.9.0
+       payload:
+         $ref: lumens.avsc'
```

Comment:
sorry, another change. Just learned how schema fragments published on exchange are referred. So modify this line as 
`$ref: /exchange_modules/83589b9b-6630-4c28-985d-174ee9172661/lumenschema/1.0.0/lumens.json'`

Reply:
Please, remember that you can add a suggestion and edit the doc yourself.
Once done, I can commit to your suggestion.
Thanks!